src/lookup.c

```diff
@@ -11,7 +11,7 @@
 #include <unistd.h>
 
 #include "libqrtr.h"
-#include "logging.h"
+#include <logging.h>
```

Comment:
So libqrtr.h is local, but logging.h is global? I think this is plain wrong. We are not installing the logging.h header, so it should be local too.

Reply:
Yes, you're right